### PR TITLE
added fontawesome-free

### PR DESCRIPTION
**Pull request by FaxriddinMaxmadiyorov**

### CODE REVIEW

**Comment by FaxriddinMaxmadiyorov** [comment]
@XaoGao Complite, check PR

File: app/javascript/application.js [hunk 1]
```diff
@@ -2,3 +2,4 @@
 import "@hotwired/turbo-rails"
 import "controllers"
 import "bootstrap"
+import "@fortawesome/fontawesome-free"
```
Comment (XaoGao) [inline app/javascript/application.js:5]: @FaxriddinMaxmadiyorov add an empty line of end file

**Review by XaoGao — COMMENTED** [review]
@FaxriddinMaxmadiyorov please fix

File: app/javascript/application.js (hunk 1 above)
Comment (FaxriddinMaxmadiyorov) [reply app/javascript/application.js]: hey, I added an empty line and pushed, could you please check?

Comment (FaxriddinMaxmadiyorov) [reply app/javascript/application.js]: @XaoGao

**Review by XaoGao — APPROVED** [review]

**Pull request merged**